packages/graphql/src/classes/authorization/Neo4jGraphQLAuthorization.ts

```diff
@@ -33,6 +33,7 @@ export class Neo4jGraphQLAuthorization {
     private authorization: Neo4jAuthorizationSettings;
 
     constructor(authorization: Neo4jAuthorizationSettings) {
+        // TODO remove this on 4.0.0
         if (authorization?.verify === false && authorization?.globalAuthentication === true) {
             throw new Neo4jGraphQLError("`globalAuthentication` option requires the `verify` option to be enabled.");
         }
```

Comment:
Why not remove this now? We need to remove the `globalAuthentication` argument if we want it configurable using directives.

Reply:
We don't need backwards compatibility, because nothing has been merged yet, so nothing is breaking. So go ahead and remove! 🙂